xml/admin_monitoring_alerting.xml

```diff
@@ -0,0 +1,1065 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"  xml:id="monitoring-alerting">
+ <title>Monitoring and Alerting</title>
+ <para>
+  In &productname; 6, &deepsea; no longer deploys a monitoring and alerting
+  stack on the &smaster;. Users have to define the &prometheus; role
+  for &prometheus; and &alertmanager;, and the &grafana; role for &grafana;.
```

Comment:
Not sure if that was just in the PDF I reviewed but `In &productname; 6,` resulted in `In SUSE Enterprise Storage 6;,` and `alertmanager` should probably be capitalized, as it is a name.

Reply:
Both of these are correct :) but &alertmanager; results in an uppercase solution: Alertmanager :)